cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
Another question: Why don't you just `#define CCM_STAR_CONF ccm_star_driver`, either in `project-conf.h` or in the `contiki-conf.h` of the platform, instead of altering this driver? The only difference is the crypto enable/restore, but the `cc2538_aes_128_driver` driver already takes care of that, so this should not be needed twice.